src/routes/docs/tooling/command-line/commands/+page.markdoc

```diff
@@ -4,65 +4,159 @@ title: Commands
+* The health command allows you to validate and monitor your Appwrite server's health.
+---
+* `migrations`
+* The migrations command allows you to migrate data between services.
+---
+* `vcs`
+* The vcs command allows you to interact with VCS providers and manage your code repositories.
 ---
-* `deploy`
-* The deploy command provides a convenient wrapper for deploying your databases, collections, functions, teams, and buckets.
+{% /table %}
+
+## Account commands {% #account-commands %}
+
+{% table %}
+* Command
+* Description
+---
+* `login [options]`
+* The login command allows you to authenticate into the CLI. This command expects the console account that you use to log into the Appwrite Console.
 ---
 * `logout`
-* The logout command allows you to logout from the CLI.
+* The logout command allows you to log out of your Appwrite account.
 ---
-* `projects`
-* The projects command allows you to view, create, and manage your Appwrite projects.
+* `register`
+* Prints link to register an Appwrite account.
 ---
-* `storage`
-* The storage command allows you to manage your project files.
+* `whoami`
+* The whomai command gives information about the currently logged-in user.
 ---
-* `teams`
-* The teams command allows you to group users of your project and to enable them to share access permissions to your project's resources.
+{% /table %}
+
+## Deployment commands {% #deployment-commands %}
+
+{% table %}
+* Command
+* Description
+---
+* `init [options]`
+* The init command provides a convenient wrapper for creating and initializing projects, functions, collections, buckets, teams, and messaging-topics in Appwrite.
+---
+* `pull`
+* The pull command helps you pull your Appwrite project, functions, collections, buckets, teams, and messaging-topics.
+---
+* `push`
+* The push command provides a convenient wrapper for pushing your functions, collections, buckets, teams, and topics.
+---
+* `run`
+* The run command allows you to run projects locally to allow easy development and quick debugging.
+---
+{% /table %}
+
+## Project commands {% #appwrite-project-commands %}
+
+{% table %}
+
+* Command
+* Description
+---
+* `account`
+* The account command allows you to authenticate and manage a user account.
 ---
 * `users`
 * The users command allows you to manage your project users.
 ---
-* `account`
-* The account command allows you to authenticate as and manage a user account.
+* `teams`
+* The teams command allows you to group users of your project and enable them to share read and write access to your project resources.
 ---
-* `avatars`
-* The avatars command provides utilities to manage images, icons, and avatars.
+* `databases`
+* The databases command allows you to create structured collections of documents and query and filter lists of documents.
 ---
 * `functions`
-* The functions command allows you view, create and manage your Appwrite Functions.
+* The functions command allows you to view, create, and manage your Cloud Functions.
```

Comment:
Let's stick with calling it Appwrite Functions. It's weird to call self-hosted Appwrite Functions "Cloud" functions. Also, we have Appwrite Cloud too so we don't want that mixed up.